code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1595,3 +1595,30 @@ var/list/rank_prefix = list(\
 /mob/living/carbon/human/proc/set_remoteview(var/atom/A)
 	remoteview_target = A
 	reset_view(A)
+
+/mob/living/carbon/human/proc/resuscitate()
+
+	if(!is_asystole() || has_organ(OP_HEART) || has_organ(BP_BRAIN))
+		return 0
+
+	if(world.time >= (timeofdeath + NECROZTIME))
+		return 0
+
+	visible_message(SPAN_NOTICE("\The [src] twitches a bit as [gender == FEMALE ? "her" : "his"] heart restarts!"))
```

Comment:
Might as well have their, instead of her/his, as it's grammatically correct

Reply:
Who their? It's one person.

Reply:
It'd be a possessive pronoun in this case, not a referral to multiple people.

Reply:
Maybe. Will change after collecting more reviews.

Reply:
Done